recipes/libcurl/all/conanfile.py

```diff
@@ -138,7 +143,7 @@ def _patch_misc_files(self):
                                   "define CURL_MAX_WRITE_SIZE 16384",
                                   "define CURL_MAX_WRITE_SIZE 10485760")
 
-        # https://github.com/curl/curl/issues/2835
+        # https://github.com/curl/curl/issues/2835  # TODO this ticket has been closed 2018, check if this is still required
```

Comment:
Since this patch and ticket are up to build infrastructure, it is better to leave patching here. Let's wait at least until apple-clang 9 is [supported by CCI](https://github.com/conan-io/conan-center-index/wiki/Supported-Platforms-And-Configurations#osx).

The upstream patch does not fix our problem because it only tweaks their Travis build machine to be a macOS 10.12 with XCode 9.2. CCI uses XCode 9.4 by the way but the underlying OS is not known.